docs/dev-guide/topolvm-csi.md

```diff
@@ -0,0 +1,40 @@
+# Topolvm CSI
+
+- LVM Operator orchestrates Topolvm CSI by creating and managing all required
+  resources for seamless usage of Logical Volumes as Persistent Volumes
+- This document outlines only the changes made to deployments of Topolvm CSI
+  components and further info please consult topolvm [docs][topolvm-docs]
+
+## CSI Driver
+
+- *csiDriver* reconcile unit deploys Topolvm CSIDriver resource
+- Ephemeral volumes support is removed (not in the code) due to below reasons:
```

Comment:
The LVM operator does not expose support for ephemeral volumes.

Please remove the section on reasons.